AppDashboard/templates/cron/view.html

```diff
@@ -0,0 +1,37 @@
+<!-- FILE:templates/cron/cron.html -->
+<!--Body content-->
+<style>
+   table {border-collapse:collapse; table-layout:fixed;}
+   table td {border:solid 5px #fab; word-wrap:break-word; border-color:gray}
+</style>
+
+<div id="content">
+    <div class="heading">
+        <h3>Cron viewer</h3>
+    </div>
+    <div class="row-fluid">
+        <div class="page-header">
+            <h1>View cron tasks:</h1>
+            <table>
+                <tr>
+                    <th>Cron job</th>
+                    <th>Description</th>
+                    <th>Frequency</th>
+                </tr>
+                {% for cron_job in cron_jobs%}
+                    <tr>
+                        <td id="command" width="30%">{{cron_job['url']}}</td>
+                        <td>{{cron_job['description']}}</td>
+                        <td>{{cron_job['frequency']}} | {{cron_job['frequency_cron_format']}}</td>
```

Comment:
I would recommend splitting this column into 2 so that the cron syntax can be labeled with a separate heading. The new heading could be something like "Generated crontab entries".